codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/AcceptorModel.java

```diff
@@ -64,7 +61,14 @@ public void setMatcher(String matcher) {
     }
 
     public String getArgument() {
-        return argument;
+        String[] unformatted = argument.split("\\.");
+        StringJoiner formatted = new StringJoiner(".");
+
+        for (String format : unformatted) {
+            formatted.add(StringUtils.uncapitalize(format));
```

Comment:
Ah ok I was trying to wrap my head around how the heck this was even working then saw this, https://github.com/aws/aws-sdk-java-v2/blob/master/jmespath-java/src/main/java/software/amazon/awssdk/jmespath/ObjectMapperSingleton.java#L28